src/routes/about-care-for-carers.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+	import Container from '$lib/components/container.svelte';
+</script>
+
+<Container>
+    <h1>About Care for Carers</h1>
+    <br>
+    <div>
+    <h2>The Space in Deal</h2>
+<br>
```

Comment:
Looking at the website these line breaks haven't left much of a gap between paragraphs. For consistency sake we should split each paragraph into its own `<p>` tag, like on the Contact Us page